data/human/news.txt

```diff
@@ -1508,13 +1605,29 @@ news "tourist pa system"
 			"Dear travelers: the spaceport is a strictly no-smoking zone. Thank you for your consideration."
 			"Dear travelers: welcome to our planet. Please remember to synchronize your personal clocks to the local time and night/day cycle duration."
 			"Please stand clear of the doors when the shuttle train is departing."
-			"This is the final boarding call for passengers Eline, John, and Robert Phillips. Please proceed to platform 3 for immediate boarding."
+			"This is the final boarding call for passengers ${tourist names}, ${tourist names}, and ${tourist names} ${tourist names}. Please proceed to platform ${platform number} for immediate boarding."
 			"Dear travelers: please take the time to adjust to normal gravity."
 		word
 			`"`
 
 
 
+phrase "republic motto graffiti"
+	word
+		`` 2
+		`Underneath are the words "PEACE AMONG THE STARS".` 2
+		`Underneath are the words "DEATH AMONG THE STARS".`
+		`Underneath are the words "TYRANNY AMONG THE STARS".`
+		`Underneath are the words "POVERTY AMONG THE STARS".`
+		`Underneath are the words "PEACE AMONG THE PARADISE WORLDS".`
+
+phrase "deep motto graffiti"
+	word
+		`` 2
+		`Underneath are the words "Peace, Prosperity, Progress".` 2
+		`Underneath are the words "Power, Poverty, Pride".`
+		`Underneath are the words "Peace, Prosperity, Progress, for those who are born lucky".`
+
```

Comment:
Where are these two phrases going to be used? Of course it will be helpful for future content but I don't see the relevance here in this PR.

Reply:
Those appear to be used on lines 1686 and 1687 in this PR already.